Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ListViewExtensions/ListViewExtensionsPage.xaml.cs

```diff
@@ -15,14 +20,22 @@ public ListViewExtensionsPage()
             this.InitializeComponent();
         }
 
+        public ICommand SampleCommand => new DelegateCommand<PhotoDataItem>(OnExecuteSampleCommand);
+
         public async void OnXamlRendered(FrameworkElement control)
         {
             var sampleListView = control.FindChildByName("SampleListView") as ListView;
 
             if (sampleListView != null)
             {
                 sampleListView.ItemsSource = await new Data.PhotosDataSource().GetItemsAsync();
+                ListViewExtensions.SetCommand(sampleListView, SampleCommand);
```

Comment:
The `.bind` file is what drives the sample in the UI, not the `.xaml` file (that's there to prevent .NET Native from optimizing out the types for the XamlReader that the sample app uses).

Therefore, it'd be better to set the DataContext for the sample to include this page, so we can bind to the SampleCommand directly vs. doing it here, this is really meant more for events.  You're probably seeing a red-herring because the binding is just failing in the sample app, and then being overwritten here anyway.

See this [example here](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MasterDetailsView/MasterDetailsViewPage.xaml.cs#L62)

Reply:
I have done this :)